### PR TITLE
Add Node 15.x support

### PR DESCRIPTION
**Pull request by Hongbo-Miao**
Fix

image

### CODE REVIEW

**Comment by Hongbo-Miao** [comment]
Added Node.js 16 too.

**Comment by jasnell** [comment]
Thank you :-) ... this gets us part way to supporting Node.js 16 but not all the way. The test suite will fail on Node.js 16 due to incompatibilities in tap, but that's a secondary issue :-). Really appreciate the PR!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [10.12.0, 10.x, 12.x, 13.x, 14.x]
+        node-version: [10.12.0, 10.x, 12.x, 13.x, 14.x, 15.x, 16.x]
```
Comment (jasnell) [inline .github/workflows/ci.yml:26]: I would remove 16.x from the test matrix for now as we know for certain the tests will not pass there yet.

Comment (Hongbo-Miao) [reply .github/workflows/ci.yml]: Yeah I adde there and wants to see if it can succeed. But I am not a member, so cannot see.
Dropped as requested.

**Review by jasnell — APPROVED** [review]

**Pull request merged**